app/Http/Requests/StoreUpdateSessionRequest.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace App\Http\Requests;
+
+use Illuminate\Foundation\Http\FormRequest;
+
+class StoreUpdateSessionRequest extends FormRequest
+{
+    /**
+     * Determine if the user is authorized to make this request.
+     *
+     * @return bool
+     */
+    public function authorize()
+    {
+        // TODO : determine of existing registration route protection is sufficient.
```

Comment:
Or create  guard that this user is entitled to change this value?

Reply:
The route  *should* already be in the auth:store group. Thus it can only fiddle with its own session. Remove. It's only there so I remember we can authorise at this point.